migrators/ternmigrator/tern.go

```diff
@@ -0,0 +1,110 @@
+package ternmigrator
+
+import (
+	"cmp"
+	"context"
+	"database/sql"
+	"errors"
+	"io/fs"
+	"os"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/tern/v2/migrate"
+
+	"github.com/peterldowns/pgtestdb"
+	"github.com/peterldowns/pgtestdb/migrators/common"
+)
+
+var _ pgtestdb.Migrator = (*TernMigrator)(nil)
+
+const defaultTableName = "public.schema_version"
```

Comment:
Why not export this (`DefaultTableName`)?

Reply:
Just because I didn't think about it and I'm generally a minimalist on exports. I exported it in my latest commit.